plugins/org.yakindu.sct.simulation.core.sexec/src/org/yakindu/sct/simulation/core/sexec/interpreter/DefaultExecutionSlotResolver.xtend

```diff
@@ -78,59 +78,80 @@ class DefaultExecutionSlotResolver implements IExecutionSlotResolver {
 		// first: get the root slot where to start the search
 		val root = (current.owner as ElementReferenceExpression).reference
 		var ExecutionSlot featureSlot = null
-		
+
 		if (root instanceof InterfaceScope) {
 			featureSlot = context.getSlot(callStack.pop.feature.fullyQualifiedName.toString)
 		} else {
 			featureSlot = resolve(context, current.owner)
-			if (featureSlot == null) {
-				return null // could not find starting slot for feature call
-			}
 		}
+
+		if (featureSlot === null) {
+			throw new IllegalStateException("Value of '"+current.feature+"' in expression '"+getFeatureCallText(e)+"' has not been set.")// could not find starting slot for feature call
+		}
+		
 		// go through all calls and traverse execution context hierarchy accordingly
 		for (FeatureCall call : callStack) {
 			featureSlot = resolveFromSlot(featureSlot, call)
 		}
 		return featureSlot
 	}
 	
+	def getFeatureCallText(FeatureCall call) {
```

Comment:
Tried 'fullyQualifiedName' alredy but gives null - is there any elegant way to get the 'feature1.anotherFeature.value' expression which could be found in the statechart itself?